samples/csharp/web/avatar/appsettings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ClientSettings": {
+    "SpeechRegion": "",
+    "SpeechKey": "",
+    "SpeechPrivateEndpoint": "",
+    "SpeechResourceUrl": "",
+    "UserAssignedManagedIdentityClientId": "",
+    "AzureOpenAIEndpoint": "",
+    "AzureOpenAIAPIKey": "",
+    "AzureOpenAIDeploymentName": "",
+    "CognitiveSearchEndpoint": "",
+    "CognitiveSearchAPIKey": "",
+    "CognitiveSearchIndexName": "",
+    "IceServerUrl": "",
+    "IceServerUrlRemote": "",
+    "IceServerUsername": "",
+    "IceServerPassword": "",
+    "DefaultTtsVoice": "en-US-JennyMultilingualV2Neural",
```

Comment:
Please remove "DefaultTtsVoice" and "EnableQuickReply" from this config file.
You can just hard code them like python code.
"DefaultTtsVoice" can be overridden from web page, and "EnableQuickReply" is rarely used.